en/components/grid_filtering.md

```diff
@@ -73,14 +99,22 @@ this.grid.clearFilter();
 
 #### Initial filtered state
 
-It is possible to set the initial filtering state of the grid by an array of expressions to the `filteringExpressions` property of the grid.
+It is possible to set the initial filtering state of the grid by an array of expressions to the `filteringExpressionsTree` property of the grid.
```

Comment:
You could remove the `initial`. The user could use this property to set grid's filtering state at any time.

Reply:
Also this sentence doesn't correspond to the sample bellow. The sentence says `filteringExpressionsTree` should contain an array of expressions which is not true. It should contain an array of `IFilteringExpressionsTree` for each column the user wants to filter.